scripts/run_project_tests.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+import subprocess
+import argparse
+import sys
+import os
+from typing import Dict, List
+
+import yaml
+from colorama import Fore, init
+
+init(autoreset=True)
+
+"""
```

Comment:
module/script docstring needs to be placed before the imports - then it will be available as __doc__ to eg. be reused as description in argparse